ibm/data_source_ibm_satellite_host_script_test.go

```diff
@@ -0,0 +1,49 @@
+/* IBM Confidential
+*  Object Code Only Source Materials
+*  5747-SM3
+*  (c) Copyright IBM Corp. 2017,2021
+*
+*  The source code for this program is not published or otherwise divested
+*  of its trade secrets, irrespective of what has been deposited with the
+*  U.S. Copyright Office. */
+
+package ibm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccIBMSatelliteAttachHostScriptDataSourceBasic(t *testing.T) {
+	locationName := fmt.Sprintf("tf-satellite-loc-%d", acctest.RandIntRange(10, 100))
```

Comment:
maintain common prefix across all testcases.. 
It would be easy to write clean up scripts

Reply:
Updated all satellite test.go files with common prefix for location creation.